hpedockerplugin/file_manager.py

```diff
@@ -549,7 +551,21 @@ def mount_share(self, share_name, share, mount_id):
         if 'status' in share:
             if share['status'] == 'FAILED':
                 LOG.error("Share not present")
-
+        fUser = None
+        fGroup = None
+        fMode = None
+        fUName = None
+        fGName = None
+        is_first_call = False
+        if share['fsOwner']:
+            fOwner = share['fsOwner'].split(':')
+            fUser = int(fOwner[0])
+            fGroup = int(fOwner[1])
+        if share['fsMode']:
+            try:
+                fMode = int(share['fsMode'])
+            except ValueError:
+                fMode = share['fsMode']
```

Comment:
why we need to assign an improper `fsMode` bits again in `fMode` .. ?
Eg. `-o fsMode=abc`, will result in `ValueError` and again "abc" will be assigned to `fMode`

Reply:
fsMode can have either Octal form or it can have value in the form of string of type:flag:permissions.

Now when user want to have directory  permissions to be set with ACL we need this value.

Reply:
In that case, can't you use type(fsMode) ? This would be more elegant rather than exception handling and detecting the type. If the type is octal (digits) then you can do the int conversion, else take as is